sync.sh

```diff
@@ -1,22 +1,60 @@
-#!/bin/sh -ex
+#!/bin/bash -e
```

Comment:
Output gets far too busy with `-x` now (could refine it further)
Note that this is now a bash, not a sh script!